src/frontend/js/types/Joanie.ts

```diff
@@ -431,6 +438,7 @@ interface OrderSubmitPayload {
 export interface PaginatedResourceQuery extends ResourcesQuery {
   page?: number;
   page_size?: number;
+  ordering?: string;
```

Comment:
```suggestion
```

Reply:
Wouldn't be possible to keep this property as it's a valid one that we want to use ?